templates/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,35 @@
-<!--
-1. Check the version of package you are using. If it's not the newest version, update and try again (see changelog while updating!).
-2. If the issue is still there, write a minimal project showing the problem and expected output.
-3. Link to the project and mention Node version and OS in your report.
+> Briefly describe the issue you are experiencing (or the feature you want to see
+added to the plugin). Tell us what you were trying to do and what happened
+instead. Remember, this is _not_ a place to ask questions. For that, go to
+http://gitter.im/webpack/webpack or https://stackoverflow.com/questions/tagged/webpack
 
-**IMPORTANT! You should use [Stack Overflow](https://stackoverflow.com/) for support related questions.**
--->
+### `📝  Details`
```

Comment:
Why do we need to format heading as code?

Reply:
### Lorem
### `Lorem`

Solely for styling purposes as 'background' similar to MDN headings 😛

Reply:
Please don’t do that if that‘s the reason — the first is much more readable, especially because of the font ;-)

Reply:
I agree with @sapegin

Reply:
Yup, better drop formatting from there.